### DISCUSSION TITLE
Is this nvidium on amd idea impossible?

### DISCUSSION THREAD

**Discussion by rikosama** [xCollateral/VulkanMod#366] [Ideas]
Does anyone know if this might work? AMD drivers older than 22.7.1, vulkan api, fabric api, sodium and nvidium.  Does anyone think that would work on amd gpus? 

So pretty much through vulkan api im thinking of a possibility of it working. Sadly my amd drivers are on 23.11 so im far too lazy to go back to test any of this out.

**Comment by ElectricLizzzard** [comment]
It will not work. All these mods are incompatible with each other. Just pick one that works best for you.
